client/src/components/RelatedItems/YourOutfit/YourOutfitCard.jsx

```diff
@@ -10,7 +11,7 @@ function YourOutfitCard({ product, removeItem }) {
     <>
       <div className="yo-item-Close-Button">
         <button type="button" onClick={() => removeItem(product.product_id)}>
-          X
+          <FaWindowClose />
```

Comment:
Ooooooh, I should use this in my modal dialogs!

I'm just using the emoji ✖️ right now :sweat_smile: